ynput/OpenPype
# Global: Do not force instance data with frame ranges of the asset

## Brief description

This aims to resolve #4317 

## Additional info

The original code that forced the frame range data into it [was added about 6 months ago](https://github.com/ynput/OpenPype/commit/43d744b24b54f21ca2f7e622ca491af91833e689). It means that before that time everything worked splendidly without that logic :) This would mean it's been there since around 3.12.2

Looking back at our earlier projects I can clearly see that published Maya models, rigs, looks etc display no frame ranges in the Loader which is what I'd expect! At some points this was broken by the above mentioned commit.

## Testing notes:

1. Publish away your favorite instances!
2. Make sure frame data is as we'd like - those that set a frame range to extract should show a frame range in the loader, those that are not time-dependent but a single frame or full scene (workfile) should not show frame ranges.
3. Make sure loading and updating (setting version) of the publish still works as expected.
